### PR TITLE
Add pontoon script

### PR DESCRIPTION
**Pull request by alicoding**
Of all the thing I added and I forgot to add this! Derp me <.<

### CODE REVIEW

File: components/pontoon.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var React = require('react');
+
+var enablePontoon = process.env.ENABLE_PONTOON === 'true';
```
Comment (alanmoo) [inline components/pontoon.jsx]: process.env.ENABLE_PONTOON isn't a boolean?

Comment (alicoding) [inline components/pontoon.jsx]: Apparently not when it's being exposed through Webpack...

Comment (alanmoo) [inline components/pontoon.jsx]: *mutters something about webpack documentation under his breath*

Comment (alanmoo) [inline components/pontoon.jsx]: I just tested this- you should just be able to drop `process.env.ENABLE_PONTOON` in line 7 and get rid of this bit and it works as expected, just like the other env vars in this repo.

Comment (alicoding) [inline components/pontoon.jsx]: Updated

**Comment by alanmoo** [comment]
R+ Pending travis

**Pull request merged**